integreat_cms/cms/forms/linkcheck/edit_url_form.py

```diff
@@ -1,9 +1,74 @@
+import logging
+
 from django import forms
+from django.core.validators import EmailValidator, URLValidator
+
+logger = logging.getLogger(__name__)
+
+
+class LinkField(forms.URLField):
+    """
+    A field for links that might be URLs but could also be mailto: or tel: links
+    """
+
+    #: Disable the default URL validator
+    default_validators = []
+    #: Whether to skip the validation URL fragments in URLField.to_python()
+    skip_url_fragment_validation = True
+
+    def to_python(self, value):
+        """
+        Convert the string value to the appropriate Pyton data strucuture for this field
+
+        :param value: The value that was input into the form
+        :type value: str
+
+        :returns: The Python value
+        :rtype: str
+        """
+        if self.skip_url_fragment_validation:
+            # Skip the URL field to_python for email and phone links
+            logger.debug(
+                "Value %r is a mailto or tel link, skipping to_python() of URLField.",
+                value,
+            )
+            value = super(forms.URLField, self).to_python(value)
+        else:
+            value = super().to_python(value)
+        return value
+
+    def clean(self, value):
+        """
+        Validate the given value and return its "cleaned" value as an
+        appropriate Python object. Raise ValidationError for any errors.
+
+        :param value: The value that was input into the form
+        :type value: str
+
+        :returns: The cleaned value
+        :rtype: str
+        """
+        if value.startswith("mailto:"):
+            email = value[7:]
+            logger.debug(
+                "Value %r is an email link, enforcing EmailValidator on %r",
+                value,
+                email,
+            )
+            self.validators.append(EmailValidator())
```

Comment:
> Enforce email validation on mailto links
I tried to add an invalid email, but it's displayed as a normal one.
Is it expected? Did I misunderstood something? 
![image](https://user-images.githubusercontent.com/115008338/230117933-25fb9c61-98fc-4ac9-a750-bde3402d7c68.png)

Reply:
Ah yeah, this was maybe a bit misleading, the email validation is only made when trying to replace an email.
Maybe this can be improved after #2207 is merged, but for now it's ok I guess.